game/scripts/npc/abilities/ember_spirit_sleight_of_fist.txt

```diff
@@ -50,7 +50,7 @@
       "04"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "creep_damage_penalty"                            "0" //OAA
+        "creep_damage_penalty"                            "45 30 15 0 0 0" //OAA
```

Comment:
this is an over nerf especially since the changes of boots. I am ok with having a creep damage penalty but how about something like 20 15 10 5 0 0?

Reply:
Changed it to "30 20 10 0". I think 20 level 1 won't be noticeable enough.